tubular-react-common/index.md

```diff
@@ -114,9 +119,12 @@ const MasterDetailRow = ({ columns, row, index }) => {
     </>
   );
 };
-{% endraw %}
 ```
+
+</div>
 <button class="nav-link link-blue" onclick="convert(this, 'usemasterdetails-hook-example-sjzwo');">Open CodeSanbox</button>
+<br />
```

Comment:
Never use <br /> to give margin between components, make a CSS class to achieve this, pls

Reply:
FTFY ``<br />``